lib/modules/manager/puppet/types.ts

```diff
@@ -0,0 +1,10 @@
+import type { SkipReason } from '../../../types';
+
+export interface PuppetfileModule {
+  name?: string;
+  version?: string;
+  tags?: Map<string, string>;
```

Comment:
i think this can be removed too because it's unused.

Reply:
same as here, we do use the git-tags to check for a newer version for this module